Dockerfile

```diff
@@ -13,20 +33,37 @@ ARG musl_sha256=a9a118bbe84d8764da0ea0d28b3ab3fae8477fc7e4085d90102b8596fc7c75e4
 ADD https://musl.libc.org/releases/${musl_src}.tar.gz /
 RUN echo ${musl_sha256} ${musl_src}.tar.gz | sha256sum -c
 
+# Build musl.
 RUN ["/bin/bash", "-c", "\
     tar xfz ${musl_src}.tar.gz && \
     cd /${musl_src} && \
     ./configure --disable-shared --prefix=/usr/local/musl && \
     make -j`nproc` && make install && make clean && \
     ln -s /usr/include/$(arch)-linux-gnu/asm /usr/include/{asm-generic,linux} /usr/local/musl/include/"]
 
-# Stage 1: install build tools + copy musl toolchain from the previous step
+# Build a recent python for gcovr.
+ENV PATH="$PATH:/opt/python3/bin"
+WORKDIR /tmp
+RUN curl -fsSL https://www.python.org/ftp/python/3.13.0/Python-3.13.0.tgz | tar xzf - && cd Python-3.13.0 && \
+    ./configure --prefix=/opt/python3/ --enable-optimizations && \
+    make -j "$(nproc)" && make -j "$(nproc)" install && \
+    cd /tmp && rm -rf /tmp/Python-*  && pip3 install gcovr
+
+# Stage 1: install build tools + copy musl toolchain and python3 from the previous step
 FROM public.ecr.aws/debian/debian:10-slim
 
-# The following command should be exactly the same as at stage 0 to benefit from caching.
-# libicu-dev is needed for the github actions runner
 RUN apt-get update && apt-get install -y --no-install-recommends \
-    sudo libicu-dev patchelf curl make g++ openjdk-11-jdk-headless && \
+        curl \
+        g++ \
+        g++-aarch64-linux-gnu \
```

Comment:
Why is cross-toolchain needed?

Reply:
Good catch, this was from a merge conflict, removing.